src/Storages/StorageURL.h

```diff
@@ -267,6 +269,22 @@ class StorageURLSink : public SinkToStorage
     bool cancelled = false;
 };
 
+static bool urlWithGlobs(const String & uri)
+{
+    return (uri.find('{') != std::string::npos && uri.find('}') != std::string::npos) || uri.find('|') != std::string::npos;
+}
+
+inline String getSampleURI(String uri, ContextPtr context)
```

Comment:
```suggestion
String getSampleURI(String uri, ContextPtr context)
```
No need for inline here